### ISSUE TITLE
proof of stake, scalability,casper and the research group

### ISSUE THREAD

**Issue by zelig** [ethereum/homestead-guide#41]
a few paragraphs and extensive resource listing 
see also https://souptacular.gitbooks.io/ethereum-tutorials-and-tips-by-hudson/content/proof-of-stake_resources.html
@homotopycolimit ?

**Comment by Souptacular** [comment]
I talked to @vladzamfir and he said nothing about PoS, stake, or casper is ready for primetime and certainly not homestead so it would be better if we do not mention things in the homestead guide that aren't happening anytime soon and are not fully ideated.

**Comment by zelig** [comment]
i disagree. This is one of the most widely watched change on the ethereum roadmap. 
We defo mention it with links to resources and the research community. 
I agree that a disclaimer should be loud and clear saying it is all in the air.

**Comment by Souptacular** [comment]
I think we should put a section with links to discussions around it, but it is so up in the air at this point, it would be detrimental to write anything definitively besides references to other sources.

Maybe something like a modified version of this:
https://souptacular.gitbooks.io/ethereum-tutorials-and-tips-by-hudson/content/proof-of-stake_resources.html

or carve out a section in the FAQ about CASPER.

**Comment by zelig** [comment]
maybe this should be somewhere around the roadmap itself. There i will put a subsection 'future directions' and mention scalability and proof of stake etc.
sure nothing definitive that list of resources and links to discussions

**Comment by void4** [comment]
A general overview of Serenity should be sufficient to prevent confusion about the Big Freeze at the end of the year :)

Currently, the Ethereum consensus uses a Proof of Work (PoW) algorithm to determine which miner gets to append a new block to the chain. The algorithm, which is called *EtHash* is defined here:
https://github.com/ethereum/wiki/wiki/Ethash

Due to inherent scalability limitations of Proof of Work algorithms, it is planned to transition the network to a Proof of Stake (PoS) based consensus system. To accomplish this, the previous Frontier release included a mining difficulty adjustment scheme, which will become noticeable by the end of the year. The network will enter a "frozen" stage, which will be used for the deployment of Proof of Stake.
